backend/server/src/args.rs

```diff
@@ -0,0 +1,13 @@
+//! This module defines the command line arguments Tobira accepts.
+
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+/// Video portal for Opencast.
+#[derive(Debug, StructOpt)]
+pub(crate) struct Args {
+    /// Path to the configuration file. If this is not specified, Tobira will
+    /// try opening `config.toml` or `/etc/tobira/config.toml`.
+    #[structopt(short, long)]
```

Comment:
Do I understand it correctly that this picks the short option for us? Could that break our interface when we add options? :thinking:

Reply:
The `short, long` means that this option can be specified via a short `-c` and a long `--config` flag. One can also say `short = "-c", long = "--config"`, but that's by default inferred from the field name.

Reply:
And this inference worries me. How does it work? Alphabetically? What if we introduce a flag `caffeine` for extra speed, later on? Then the interface silently breaks, right?

Reply:
Introducing another options starting with `c` will error. Unfortunately, it will only error at runtime right now, but it *always* panics even before parsing the args. So no silent breaking.

> How does it work? Alphabetically?

Not sure what you mean. `short` takes the first unicode char of the option. `long` also converts `snake_case` to `--kebab-case`.

Reply:
> Not sure what you mean. short takes the first unicode char of the option. long also converts snake_case to --kebab-case.

I meant when you don't specify anything.

But ok, when it errors, I'm totally fine with keeping this implicit. :D